.github/workflows/node_modules-pull_request.yml

```diff
@@ -0,0 +1,67 @@
+name: node_modules
+on: pull_request
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    outputs:
+      package_json: ${{ steps.package_json.outputs.result }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Read package.json
+        id: package_json
+        uses: actions/github-script@v3
+        with:
+          script: return require('fs').readFileSync('package.json');
+          result-encoding: string
```

Comment:
Any reason why this is better than `run: cat package.json` ?

Reply:
The goal here is to get the content of `package.json` into the `.result` variable for the step.

Reply:
ah, and you don't want to use the `::set-output` stuff, ack.